### ISSUE TITLE
ko.mapping.defaultOptions().include should be an array or string.

### ISSUE THREAD

**Issue by arctidog** [SteveSanderson/knockout.mapping#67]
2.1.0 was fine, but 2.1.1 causes an exception when running the following snippet:

```
        ko.mapping.fromJS({
            "firstName": "fjidwj",
            "lastName": "gjfigjp",
            "company": "",
            "homePhone": "",
            "mobilePhone": "32456754356",
            "workPhone": "",
            "faxPhone": "",
            "primaryPhoneNumberName": "Mobile",
            "bestTimeToReach": "Evening",
            "emailAddress": "gjiwogjw@hjiwofgjhw.com",
            "emailAddressReenter": "gjiwogjw@hjiwofgjhw.com",
            "__ko_mapping__": {
                "include": [
                    "_destroy"
                ],
                "ignore": [],
                "copy": [],
                "mappedProperties": {
                    "firstName": true,
                    "lastName": true,
                    "company": true,
                    "homePhone": true,
                    "mobilePhone": true,
                    "workPhone": true,
                    "faxPhone": true,
                    "primaryPhoneNumberName": true,
                    "bestTimeToReach": true,
                    "emailAddress": true,
                    "emailAddressReenter": true
                }
            }
        });
```

**Comment by sagacity** [comment]
Thanks for the repro. This is related to issue #63.

**Comment by DzenisevichK** [comment]
Why do you pass a object with internal `__ko_mapping__` as the **first** argument? All properties will be wrapped in observable and version 2.1.0 on `toJS` should also throw exception if I am rigth. Difference between 2.1.1 and 2.1.0 related to this issue is only that 2.1.1 checks options in `fromJS` too.

**Comment by sagacity** [comment]
@arctidog Why are you passing in 'ko_mapping' like that?

**Comment by arctidog** [comment]
The application never directly deals with it, but we have a few to/from conversions that make that appear in a piece of data. I honestly don't know why it's there.

**Comment by kijanawoodard** [comment]
@arctidog I think I just ran into this problem myself. In my case, I was using ko.mapping.fromJSON and then ko.toJSON. If you then try ko.mapping.fromJSON again it doesn't work. ko.toJSON and toJS leave the **ko mapping** stuff in there. I called ko.mapping.toJSON and got what I was looking for.

Also I got around the original problem by making sure I added extra properties to my source js object before mapping to a ko object.
